base-template-wrapper.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+# Base Git hook template from https://github.com/rycus86/githooks
+#
+# It allows you to have a .githooks folder per-project that contains
+# its hooks to execute on various Git triggers.
+
+GITHOOKS_RUNNER=$(git config --global githooks.runner)
+
+if [ ! -x "$GITHOOKS_RUNNER" ]; then
+    echo "! Githooks runner points to non existing location:" >&2
+    echo "   \`$GITHOOKS_RUNNER\`" >&2
+    echo "  or it is not executable!" >&2
+    echo " Please run the Githooks install script again to fix it." >&2
+    exit 1
+fi
+GITHOOKS_HOOK_NAME=$(basename "$0") \
+GITHOOKS_HOOK_FOLDER=$(dirname "$0") \
```

Comment:
I'd consider just passing `$0` here and letting the runner script separate it into hook name and hook folder, to make this script even simpler and make it easier to change things later.

Reply:
yeah totally

Reply:
Lets see, it took some iteration till I arrived here, with tests now passing I can try this:-)

Reply:
Also I use exec now.

Reply:
Done!